docs/source/frequently_asked_questions.ipynb

```diff
@@ -620,6 +620,66 @@
+    "\n",
+    "- Use `include_variables` to specify the only variables in an entity that should be used to create features for that primitive. It is a dictionary mapping an entity id to a list of variable names to include.\n",
+    "\n",
+    "You can also use `primitive_options` to specify which entities or variables you wish to use as groupbys for groupby transformation primitives:\n",
+    "\n",
+    "- Use `ignore_groupby_entities` to specify entities that should not be used to get groupbys for that primitive. It is a list of entity ids to ignore.\n",
+    "\n",
+    "- Use `include_groupby_entities` to specify the only entities that should be used to get groupbys for that primitive. It is a list of entity ids to include.\n",
+    "\n",
+    "- Use `ignore_groupby_variables` to specify variables in an entity that should not be used as groupbys for that primitive. It is a dictionary mapping an entity id to a list of variable names to ignore.\n",
+    "\n",
+    "- Use `include_groupby_variables` to specify the only variables in an entity that should be used as groupbys for that primitive. It is a dictionary mapping an entity id to a list of variable names to include.\n",
+    "\n",
+    "Here is an example of using some of these options:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es = ft.demo.load_mock_customer(return_entityset=True)\n",
+    "\n",
+    "feature_matrix, feature_defs = ft.dfs(entityset=es,\n",
+    "                                      target_entity=\"customers\",\n",
+    "                                      primitive_options={\"mode\": {\"ignore_entities\": [\"sessions\"],\n",
+    "                                                                  \"include_variables\": {\"products\": [\"brand\"],\n",
+    "                                                                                        \"transactions\": [\"product_id\"]}},\n",
+    "                                                         # For mode, ignore the \"sessions\" entity and only include \"brands\" in the\n",
+    "                                                         # \"products\" entity and \"product_id\" in the \"transactions\" entity\n",
+    "                                                         (\"count\", \"mean\"): {\"include_entities\": [\"sessions\", \"transactions\"]}\n",
+    "                                                         # For count and mean, only include the entities \"sessions\" and \"transactions\"\n",
+    "                                                         })\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**For a more examples of specifying options for DFS, please visit:**\n",
+    "\n",
+    "- [Specifying Primitive Options](https://docs.featuretools.com/guides/specifying_primitive_options.html)"
```

Comment:
does it work to make this just a relative link? `/guides/specifying_primitive_options.html`?

Reply:
It does. I was following the same format as the "Improve Computational Performance" link in the "How Do I Speed Up Runtime of DFS?" section, but can change it if the relative link is preferred.

Reply:
ah, we should probably make them relative there. we also host docs at http://featuretools.featurelabs.com for example